Clover/app/src/main/java/org/floens/chan/core/model/orm/Filter.java

```diff
@@ -49,6 +50,12 @@
     @DatabaseField(canBeNull = false)
     public int color;
 
+    @DatabaseField(canBeNull = false)
```

Comment:
I meant to point to this line of code, sorry.

Reply:
Done.